InteractionBase/src/commands/Command.cpp

```diff
@@ -25,13 +25,14 @@
 ***********************************************************************************************************************/
 
 #include "commands/Command.h"
+#include "precompiled.h"
 
 namespace Interaction {
 
-Command::~Command()
+Command::Command(QString name, bool canBeUsedInMenu)
```

Comment:
Why did you remove the destructor of Command? We need it here even if it's empty since it's important to put the destructors of virtual base classes in a cpp file, in order to indicate where the virtual table will be. Please bring it back.